crates/bdk/src/descriptor/template.rs

```diff
@@ -79,7 +79,9 @@ impl<T: DescriptorTemplate> IntoWalletDescriptor for T {
 ///
 /// let key =
 ///     bitcoin::PrivateKey::from_wif("cTc4vURSzdx6QE6KVynWGomDbLaA75dNALMNyfjh3p8DRRar84Um")?;
-/// let mut wallet = Wallet::new_no_persist(P2Pkh(key), None, Network::Testnet)?;
+/// let key2 =
```

Comment:
Isn't it better to give better names than `key` and `key2`?

Reply:
How about `key_a`, `key_b`

Reply:
You've used `descriptor` and `descriptor_change` in the PR somewhere.
I like that. It is informative.
Because here `key` is the receiving/external "address type",
and `key2` is the change/internal "address type.

Reply:
Readers of the code will see `key` is a bitcoin private key.

To expand on that, here `key` is being used to build a descriptor "template" which can then be passed to `Wallet::new_no_persist`, as all `DescriptorTemplate` implementors are also `IntoWalletDescriptor`.

Reply:
`key_external` and `key_internal`?